great_expectations/render/renderer/email_renderer.py

```diff
@@ -80,3 +99,9 @@ def _get_report_element(self, docs_link):
         else:
             logger.warning("No docs link found. Skipping data docs link in the email message.")
         return report_element
+
+    def _build_report_element_block(self, validation_result_urls: list[str]) -> str | None:
+        for url in validation_result_urls:
+            fixed_url = pathlib.Path(urllib.parse.unquote(url)).as_posix()
```

Comment:
I'm confused by this step. It looks like the net result is going from `file:///myfile` to `file:/myfile`. My understanding is that the former is correct. Why not just pass through the url that the user configured?

Reply:
You're right, the first part is unnecessary! Unquoting, however, is needed to parse the urls that are supplied in format `... /gx/uncommitted/data_docs/local_site/validations%5Ccompanies%5Ctestsuite%5C ...`